### PR TITLE
fix: embeddedListSearch component pagination css fixes

### PR DESCRIPTION
**Pull request by ShubhamThakre**
## Checklist
- embeddedListSearch component pagination css fixes on glossaryTerm, Domain, Container, Users and Groups screen. 
Screenshot 2022-04-12 at 8 51 07 PM
Screenshot 2022-04-12 at 8 50 35 PM

@jjoyce0510

### CODE REVIEW

File: datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearchResults.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const FiltersContainer = styled.div`
 
 const ResultContainer = styled.div`
     flex: 1;
-    margin-bottom: 20px;
+    // margin-bottom: 20px;
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearchResults.tsx:29]: we can remove this then

File: datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ import { useGetSearchResultsForMultipleQuery } from '../../../../../../graphql/s
 import { GetSearchResultsParams, SearchResultsInterface } from './types';
 
 const Container = styled.div`
-    overflow: scroll;
+    // overflow: scroll;
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx:20]: we can remove this

File: datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearchResults.tsx (hunk 1 above)
Comment (ShubhamThakre) [reply datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearchResults.tsx:29]: removed.

File: datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx (hunk 2 above)
Comment (ShubhamThakre) [reply datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx:20]: removed

**Pull request closed without merging**